packages/@aws-cdk/aws-ecs/lib/asset-image.ts

```diff
@@ -0,0 +1,232 @@
+import cfn = require('@aws-cdk/aws-cloudformation');
+import iam = require('@aws-cdk/aws-iam');
+import lambda = require('@aws-cdk/aws-lambda');
+import cdk = require('@aws-cdk/cdk');
+import cxapi = require('@aws-cdk/cx-api');
+import fs = require('fs');
+import path = require('path');
+import { ContainerDefinition } from './container-definition';
+import { IContainerImage } from './container-image';
+
+export interface AssetImageProps {
+  /**
+   * The directory where the Dockerfile is stored
+   */
+  directory: string;
+}
```

Comment:
I guess we could also allow people to configure the name of `Dockerfile`

Reply:
You mean as an additional parameter?

Reply:
Yap